File: Contest2/Orange/print-the-elements-of-a-linked-list-in-reverse.py
def reversePrint(head):
    arr=list()
    curr=head
    if (head == None):
        return
    else:
        while(True):
            arr.append(curr.data)
            if(curr.next==None):
                break
            else:
                curr=curr.next
    arr=arr[::-1]
    x="\n".join(str(e) for e in arr)
    print(x)

File: Contest2/Orange/print-the-elements-of-a-linked-list-in-reverse.cpp
// Complete the reversePrint function below.
/*
 * For your reference:
 *
 * SinglyLinkedListNode {
 *     int data;
 *     SinglyLinkedListNode* next;
 * };
 *
 */
void reversePrint(SinglyLinkedListNode* head) { 
    if(head==NULL)
        return;
    else
        {
          reversePrint(head->next);
          cout<<head->data<<endl;
        }

}
